Resources/Prototypes/Traits/mental.yml

```
@@ -41,12 +41,12 @@
     - !type:TraitReplaceComponent
       components:
         - type: PotentiaModifier
-          potentiaMultiplier: 1.25
+          potentiaMultiplier: 2
```

```suggestion
          potentiaMultiplier: 1.5
```
multiplier of 2 is INSANELY strong yo.

---

I was considering either this or lowering the potentia needed per spell slot used.